Embedded Metadata.js

```diff
@@ -680,11 +680,26 @@ function getAuthorFromByline(doc, newItem) {
 	}
 	
 	if(actualByline) {
+		//Z.debug(actualByline.innerHTML)
 		byline = ZU.trimInternal(actualByline.textContent);
-		Z.debug("Extracting author(s) from byline: " + byline);
-		byline = byline.split(/\bby[:\s]+/i);
-		byline = byline[byline.length-1].replace(/\s*[[(].+?[)\]]\s*/g, '');
-		var authors = byline.split(/\s*(?:(?:,\s*)?and|,|&)\s*/i);
+		var authors = [];
+		//make sure we're separating multiple authors on same line in byline
+		if (ZU.xpath(actualByline, './/*[string-length(text())>1]').length > 2){
```

Comment:
wouldn't this be a problem if, say first name and last name for each author are styled differently? (E.g. Last name in all-caps) this would require name parts to be in separate elements and that shouldn't work with this code 

IIRC, the original reasoning was that if there are going to be multiple authors per byline, they would be separated somehow (comma or and). I didn't take into account implicit separators via li elements. I think we should use that structure for splitting up authors, since that also makes sense semantically, instead of something on random elements.